molecule/default/molecule.yml

```diff
@@ -20,11 +20,8 @@ platforms:
       - name: cartridge-network
     purge_networks: true
   - name: vm2
-    # XXX: wait for Tarantool fix
```

Comment:
What the fix?

Reply:
This fix https://github.com/tarantool/tarantool/issues/4606.